controller/src/main/java/org/jboss/as/controller/services/path/PathResourceDefinition.java

```diff
@@ -134,6 +133,7 @@ public void registerAttributes(ManagementResourceRegistration resourceRegistrati
         SimpleAttributeDefinition pathAttr = specified ? PATH_SPECIFIED : PATH_NAMED;
         resourceRegistration.registerReadWriteAttribute(pathAttr, null, new PathWriteAttributeHandler(pathManager, pathAttr, services));
         resourceRegistration.registerReadOnlyAttribute(READ_ONLY, null);
+        PathSizeMetricHandler.registerMetric(resourceRegistration, pathManager, PATH, RELATIVE_TO);
```

Comment:
I see you partially address my concern in PathSizeMetricHandler.registerMetric by skipping server-config.